circle.yml

```diff
@@ -0,0 +1,52 @@
+# https://circleci.com/gh/spyder-ide/spyder-vim/
+
+machine:
+  environment:
+    # Python versions to tests (Maximum of 4 different versions)
+    # The last container is used to test with pyqt5 wheels
+    PY_VERSIONS: "2.7 3.6 3.5"
+    # For Coveralls
+    COVERALLS_REPO_TOKEN: sTGxv5iQxuiBQCoxLB53t6m2xSe58rAIq
```

Comment:
This token should be deleted form here (and regenerated), and added instead to https://circleci.com/gh/spyder-ide/spyder-vim/edit#env-vars